features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js

```diff
@@ -322,7 +334,37 @@ class BasicInfo extends Component {
                                     <IconButton aria-label="Add to favorites">
                                         <FavoriteIcon/>
                                     </IconButton>
-                                    <IconButton aria-label="Share">
+                                    <IconButton aria-label="Share" onClick={this.handlePopupOpen}>
+                                        <Popover
+                                            open={this.state.openPopup}
+                                            anchorEl={this.state.anchorEl}
+                                            onClose={this.handlePopupClose}>
+                                            <div id="share_div_social" className="share_dives">
+                                                {/* Facebook */}
+                                                <a className="social_links" id="facebook"
+                                                   href="http://www.facebook.com/sharer.php?u=https%3A%2F%2F172.17.0.1%3A9444%2Fstore%2Fapis%2Finfo%3Fname%3Dfoo%26version%3D1.0.0%26provider%3Dadmin"
```

Comment:
sharing URL cannot be static. Need to map correct URLs

Reply:
This is fix with: https://github.com/wso2/carbon-apimgt/pull/5284/